tests/testthat/test-dbz_eta.R

```diff
@@ -1,8 +1,24 @@
 test_that("dbz_to_eta() returns error on incorrect parameters", {
+  test_that("dbz_to_eta() returns error on incorrect parameters", {
+    expect_error(dbz_to_eta('not_a_dbz', 10))
```

Comment:
You can also indicate what error message the user will get back: 

```R
expect_error(dbz_to_eta('not_a_dbz', 10), "'dbz' must be a strictly positive numeric.", fixed = TRUE)
```

If you drop the `fixed = TRUE` you can write part of the error message that needs to match:

```R
expect_error(dbz_to_eta('not_a_dbz', 10), "'dbz' must be")
```

Reply:
Ok, nice!  should be resolved with 3f45a0f743dd35f99641b8bc71cc8c576317f034